docs/install.md

```diff
@@ -28,10 +28,8 @@ pnpm add --save-dev --save-exact prettier
 
 Then, create an empty config file to let editors and other tools know you are using Prettier:
 
-<!-- Note: `echo "{}" > .prettierrc.json` would result in `"{}"<SPACE>` on Windows. The below version works in cmd.exe, bash, zsh, fish. -->
```

Comment:
Might be useful to keep some kind of comment around in case someone opens a PR to “simplify” this to using `echo` again.

Reply:
https://github.com/prettier/prettier/pull/15367/commits/51effd971dea63958eb4c2a062bebcbf1e246548